test/e2e/serial/workload_placement_test.go

```diff
@@ -1657,6 +1667,231 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 				return ok
 			}, time.Minute, time.Second*5).Should(BeTrue(), "resources not restored on %q", targetedNodeName)
 		})
+
+		It("[test_id:47674][reboot_required][slow] should be able to modify the configurable values under the NUMAResourcesOperator and NUMAResourcesScheduler CR", func() {
+
+			nroSchedObj := &nropv1alpha1.NUMAResourcesScheduler{}
+			err := fxt.Client.Get(context.TODO(), client.ObjectKey{Name: nrosched.NROSchedObjectName}, nroSchedObj)
+			Expect(err).ToNot(HaveOccurred(), "cannot get %q in the cluster", nrosched.NROSchedObjectName)
+			initialNroSchedObj := nroSchedObj.DeepCopy()
+
+			nroOperObj := &nropv1alpha1.NUMAResourcesOperator{}
+			err = fxt.Client.Get(context.TODO(), client.ObjectKey{Name: objects.NROName()}, nroOperObj)
+			Expect(err).ToNot(HaveOccurred(), "cannot get %q in the cluster", objects.NROName())
+			initialNroOperObj := nroOperObj.DeepCopy()
```

Comment:
we already to this in `test_suite_serial_test.go` but this test wants to mutate these values, so it's safer to redo that here.